assignments/objects.js

```diff
@@ -10,29 +10,71 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
+/*const example = {
   "id": 0,
   "name": "Example",
   "email": "examples@you.edu",
   "gender": "F"
-}
+}*/
 
 // Write your intern objects here:
 
-
+const Mitzi = {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F",
+}
+console.log(Mitzi["name"])
+const Kennan = {
+    "id": 2,
+    "name": "Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M",
+    speak: function(){
+        return "Hello, my name is Kennan!";
+    }
+}
+console.log(Kennan.speak())
+console.log(Kennan["id"])
+const Keven = {
+    "id": 3,
+    "name": "Keven",
+    "email": "kmummery2@wikimedia.org",
+    "gender": "M",
+}
+console.log(Keven["email"])
+const Gannie = {
+    "id": 4,
+    "name": "Gannie",
+    "email": "gmartinson3@illinois.edu",
+    "gender": "M",
+}
+console.log(Gannie["name"])
+const Antonietta = {
+    "id": 5,
+    "name": "Antonietta",
+    "email": "adaine5@samsung.com",
+    "gender": "F",
+    multiplyNums: function(a,b){
+        return a*b;
+    }
+}
+console.log(Antonietta["gender"])
+console.log(Antonietta.multiplyNums(3,4))
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+//console.log(Mitzi["name"])
 // Kennan's ID
-
+//console.log(Kennan["id"])
 // Keven's email
-
+//console.log(Keven["email"])
 // Gannie's name
-
+//console.log(Gannie["name"])
 // Antonietta's Gender
-
+//console.log(Antonietta["gender"])
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
```

Comment:
Capitalization matters. If you capitalize the names in these console logs, you will find that it would work.

With that said, the convention is that your variable names should be "camel-cased". Camel-case means that the first letter is _always_ lowercase. Any subsequent words would have its first letter capitalized:

`aVariableName`
`num`
`anotherParamter`

Generally, you capitalize the first letter of the word only if it's a class, or a constructor function which generates instanced objects (_which a class basically boils down into_). You'll learn more about these things in future lessons.